core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java

```diff
@@ -41,6 +41,8 @@ public NewSizeOperation(InputSocket.Factory inputSocketFactory,
     this.heightSocket = inputSocketFactory.create(heightHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```

Comment:
__DO NOT DO THIS__

Use a builder for the socket hint to define a default value.